### ISSUE TITLE
Information on select properties/predicates for subject conversion work

### ISSUE THREAD

**Issue by arwenhutt** [ucsdlib/damsmanager#172]
### Summary
I'm trying to identify where we can/want to put some of the information from the subject conversion project, and need to know if we are currently using a few properties/predicates [1], if so how/where, and whether they can be used in our main authority record classes.

**For each of the following properties/predicates:**
- mads:hasCloseExternalAuthority
- mads:hasBroaderExternalAuthority
- mads:hasHiddenVariant
- mads:hasHiddenLabel
- mads:alternativeLabel
- mads:definitionNote

**Can you tell me:**
1. Have they been used in the DAMS 
2. If they have been used, a URI for the record they are in (or other context)
3. Can they currently be used in mads:CorporateName, mads:PersonalName, mads:FamilyName, mads:Geographic, dams:ScientificName, mads:Temporal, mads:Topic

[1] I identified the properties in this file https://github.com/ucsdlib/damsrepo/blob/12dedb0651c626556acd7ad366c62a6030c9929d/src/sample/predicates/predicates.nt

**Comment by lsitu** [comment]
@arwenhutt Attached is the report on the using of property/predicate mads:hasCloseExternalAuthority  in DAMS 4 with fields DAMS ARK, Value Used, Subject Type, Predicate, Subject Label, Example, Object Count:
[mads_hasCloseExternalAuthority.txt](https://github.com/ucsdlib/damsmanager/files/1330887/mads_hasCloseExternalAuthority.txt)

I also found that ComplexSubject Aquaculture--Korea (North)--Posters (http://library.ucsd.edu/ark:/20775/bb40629424) has mads:Topic Aquaculture (http://lib-hydratail-prod.ucsd.edu:8080/dams/api/objects/bb01721196) that includes the use of mads:hasCloseExternalAuthority with value https://en.wikipedia.org/wiki/Aquaculture. Here are those two objects that link to ComplexSubject Aquaculture--Korea (North)--Posters:
https://library.ucsd.edu/dc/object/bb12639869
https://library.ucsd.edu/dc/object/bb13322440

It looks like we haven't created any predicates for mads:hasHiddenLabel and
mads:alternativeLabel in damsrepo yet. And I can't find any examples of using properties/predicates mads:hasBroaderExternalAuthority, mads:hasHiddenVariant and mads:definitionNote.

For your questions:
1. Have they been used in the DAMS
A: Only mads:hasCloseExternalAuthority is used in DAMS 4 at this time.

2. If they have been used, a URI for the record they are in (or other context)
A: See the "Value Used" field in the report mads_hasCloseExternalAuthority.txt above.

3. Can they currently be used in mads:CorporateName, mads:PersonalName, mads:FamilyName, mads:Geographic, dams:ScientificName, mads:Temporal, mads:Topic
A: Yes. The report shows that property mads:hasCloseExternalAuthority is used in mads:CorporateName, mads:Geographic, dams:ScientificName, mads:Topic, and mads:GenreForm.

Please let me know if I miss anything in the report. Thanks.

**Comment by arwenhutt** [comment]
Thanks @lsitu 
To make sure I understand correctly, can we use all of the properties in all of the authority record types we have (question 3 in the description)?

**Comment by lsitu** [comment]
@arwenhutt I think we can but this is your call for it. And I think how to expose those properties is another issue since we haven't take advantage of those properties at this time at all.

**Comment by arwenhutt** [comment]
Basically I'm trying to identify the best temporary holding places (predicates) for some of the data from the subject conversion project.  Ideally the ones we use wouldn't be used yet (or at least in a different way) or require any/much development work.  I think using the predicates will come later, and maybe not until D5.

**Comment by arwenhutt** [comment]
@lsitu 
Is there any work that needs to be done in the system to do the following?

* use all three of these properties:
    - mads:hasHiddenVariant
    - mads:hasBroaderAuthority
    - mads:hasCloseExternalAuthority (values consistent with current dams usage)

* in each of these authority records types:
    - mads:CorporateName
    - mads:PersonalName
    - mads:FamilyName
    - mads:Geographic
    - dams:ScientificName
    - mads:Temporal
    - mads:Topic

* for storing metadata but not exposing them in the public

**Comment by lsitu** [comment]
@arwenhutt Maybe we just need to update the Subject Import tool in damsmanager to support them.

**Comment by arwenhutt** [comment]
@lsitu Right now we are focused on adding them as part of the subject conversion project, so not an immediate requirement for updating the ingest tools, but I will add creating an issue to add these to the ingest tool to our "things to do" list.  Thanks!
